Assets/HoloToolkit/Sharing/Tests/Scripts/RemoteHeadManager.cs

```diff
@@ -137,7 +138,7 @@ private void UpdateHeadTransform(NetworkInMessage msg)
             Quaternion headRot = CustomMessages.Instance.ReadQuaternion(msg);
 
             RemoteHeadInfo headInfo = GetRemoteHeadInfo(userID);
-            headInfo.HeadObject.transform.localPosition = headPos;
+            headInfo.HeadObject.transform.localPosition = headPos += new Vector3(0,-1.0f,0);
```

Comment:
What does the document say when you hover over the Vector3 variable and see what is displayed in the tool tip? I feel that the parameters of the Vector3 class are taking in parameters that are (x, y, z) coordinates on a plane. That means that the "-1.0f" float value is the middle value which is the y and therefore (according to physics) denotes the up/down position. So you are correct in assuming that it is 1 meter below (with the "-" indicating a negative value below 0). 

Research indicates (https://docs.unity3d.com/ScriptReference/Vector3.html) that "below" is "Shorthand for writing Vector3(0, -1, 0)." in simple integer notation.